nin/dasBoot/FullscreenRenderTargetPool.js

```diff
@@ -0,0 +1,39 @@
+class FullscreenRenderTargetPool {
+  constructor() {
+    this.renderTargets = [];
+    this.used = 0;
+  }
+
+  _createFullscreenRenderTarget() {
+    const renderTarget = new THREE.WebGLRenderTarget(16 * GU, 9 * GU, {
+      height: 16 * GU,
+      width: 16 * GU,
+      minFilter: THREE.LinearFilter,
+      magFilter: THREE.LinearFilter,
+      format: THREE.RGBAFormat,
+    });
+    return renderTarget;
+  }
+
+  getFullscreenRenderTarget() {
+    let renderTarget = this.renderTargets[this.used];
+    if(!renderTarget) {
+      renderTarget = this._createFullscreenRenderTarget();
+      this.renderTargets[this.used] = renderTarget;
+    }
+    this.used++;
+    return renderTarget;
+  }
+
+  withdrawFullscreenRenderTargets() {
+    this.used = 0;
+  }
+
+  resize() {
+    for(let i = 0; i < this.renderTargets.length; i++) {
+      this.renderTargets[i].setSize(16 * GU, 9 * GU);
```

Comment:
We might want to take custom aspect ratios into account https://github.com/ninjadev/nin/pull/490

Reply:
good idea